problems/for-loop/problem.md

```diff
@@ -29,6 +29,8 @@ On each iteration of the loop, add the number `i` to the `total` variable. To do
 total += i;
 ```
 
+When this statement is used in a for loop, it can also be known as _an accumulator_.  Think of it like a cash register's running total while each item is scanned and added up.  For this challenge, you have 10 items and they just happen to be increasing in price by 1 each item (with the first item free!).
```

Comment:
I think, translating an "accumulator" in Japanese is difficult and “accumulator” is a little difficult for computer science newcomers.

It may be no necessary to mention "accumulator". 🤔

Reply:
I don't know about the translation but maybe "running total" would work better?

Reply:
Perhaps omitting the first sentence and add the following instead, since students seem to feel like this is incrementing `total`?  I figured "accumulator" might be a bit much.

> This may seem similar to `total++;`, but it is not. Think of it [. . .]